mobile/lib/modules/backup/background_service/background.service.dart

```diff
@@ -364,30 +362,17 @@ class BackgroundService {
       );
       if (backupOk) {
         await Store.delete(StoreKey.backupFailedSince);
-        final backupAlbums = [...selectedAlbums, ...excludedAlbums];
-        backupAlbums.sortBy((e) => e.id);
-        db.writeTxnSync(() {
-          final dbAlbums = db.backupAlbums.where().sortById().findAllSync();
-          final List<int> toDelete = [];
-          final List<BackupAlbum> toUpsert = [];
-          // stores the most recent `lastBackup` per album but always keeps the `selection` from the most recent DB state
-          diffSortedListsSync(
-            dbAlbums,
-            backupAlbums,
-            compare: (BackupAlbum a, BackupAlbum b) => a.id.compareTo(b.id),
-            both: (BackupAlbum a, BackupAlbum b) {
-              a.lastBackup = a.lastBackup.isAfter(b.lastBackup)
-                  ? a.lastBackup
-                  : b.lastBackup;
-              toUpsert.add(a);
-              return true;
-            },
-            onlyFirst: (BackupAlbum a) => toUpsert.add(a),
-            onlySecond: (BackupAlbum b) => toDelete.add(b.isarId),
-          );
-          db.backupAlbums.deleteAllSync(toDelete);
-          db.backupAlbums.putAllSync(toUpsert);
-        });
+        // TODO: update album specific last backup time
```

Comment:
this TODO needs to be resolved

Reply:
Yes, but I'm planning to do it in a successive PR. Adding in the TODO just in case I forget to do it.